pyiron_base/database/filetable.py

```diff
@@ -636,3 +657,11 @@ def get_job_status_from_file(hdf5_file, job_name):
         return read_hdf5(hdf5_file, job_name + "/status")
     else:
         return None
+
+
+def get_most_common_path(path, reference_paths):
```

Comment:
Unless we explicitly expect this to get used somewhere else, I'd just make it private for now. If someone finds it and wants it, we can always make it public later.

Reply:
Agreed - I made it a private function. In the mid-term I would like to move it to a more general place as we do something similar when matching the current directory to a project path. Still that can be done in a separate pull request.